File: solutions/csharp/raindrops/1/Raindrops.cs
using System.Text;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var builder = new StringBuilder();
        if (number % 3 == 0) builder.Append("Pling");
        if (number % 5 == 0) builder.Append("Plang");
        if (number % 7 == 0) builder.Append("Plong");
        if (builder.Length == 0) return $"{number}";
        return builder.ToString();        
    }
}

File: solutions/javascript/raindrops/1/raindrops.js
export const convert = (number) => {
  let result = "";
  for (const [key, value] 
       of Object.entries({3: "Pling", 5: "Plang", 7: "Plong"})
      ) {
    if (!(number % key)) result += value;
  }
  return !result ? number.toString() : result;
};
